x/ccv/consumer/keeper/changeover.go

```diff
@@ -5,17 +5,30 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
+// ChangeoverIsComplete returns whether the standalone to consumer changeover process is complete.
+func (k Keeper) ChangeoverIsComplete(ctx sdk.Context) bool {
+	if !k.IsPrevStandaloneChain() {
+		panic("ChangeoverIsComplete should only be called on previously standalone consumers")
+	}
+	return ctx.BlockHeight() >= k.FirstConsumerHeight(ctx)
+}
+
+// The first height that the ccv valset will be in effect is 2 blocks after init genesis height
+// (aka height that the ccv module first returned updates to tendermint), because if init genesis is block N,
+// the new valset is committed in block N+1, and in effect for block N+2.
+func (k Keeper) FirstConsumerHeight(ctx sdk.Context) int64 {
+	return k.GetInitGenesisHeight(ctx) + 2
```

Comment:
Use `ValidatorUpdateDelay+1`, see https://github.com/cosmos/cosmos-sdk/blob/c0fe4f7da17b7ec17d9bea6fcb57b4644f044b7a/types/staking.go#L18 and https://github.com/cosmos/cosmos-sdk/blob/c0fe4f7da17b7ec17d9bea6fcb57b4644f044b7a/x/slashing/keeper/infractions.go#L85.